### ISSUE TITLE
Typo in HighSpeedCraftHarzardous

### ISSUE THREAD

**Issue by lderks** [tbsalling/aismessages#28]
There is a typo in `ShipType.java`. `HighSpeedCraftHarzardous` should be `HighSpeedCraftHazardous` without the `r`.

https://github.com/tbsalling/aismessages/blob/master/src/main/java/dk/tbsalling/aismessages/ais/messages/types/ShipType.java#L35
```
HighSpeedCraftHarzardousA(41)
HighSpeedCraftHarzardousB(42)
HighSpeedCraftHarzardousC(43)
HighSpeedCraftHarzardousD(44)
```

**Comment by tbsalling** [comment]
Hi Leon - thanks. I'll get that fixed. If you want to get proper credit as a contributor you are welcome to send me a pull request with the fix - then I shall merge it. Otherwise I'll look into it sometime tomorrow.

**Comment by tbsalling** [comment]
Fixed by commit 3cf1e746e61e37a451bcf52bd2b61947e4d95035
